install_mac.py

```diff
@@ -145,73 +152,96 @@ def install_brew_packages(disable: List[str]):
 
             brew('install', '--cask', cask, '--force')
 
-    logger.info('installing blacktop\'s ipsw')
-    brew('install', 'blacktop/tap/ipsw')
+    confirm_install('install ipsw', brew['install', 'blacktop/tap/ipsw'])
 
-    logger.info('setting default browser to google chrome')
-    local['defaultbrowser']('chrome')
+    if 'google-chrome' not in brew_list and 'defaultbrowser' in brew_list:
+        brew_list = brew('list').split('\n')
```

Comment:
use the already existing `brew_list`

Reply:
this line comes after the prompting section so need to do brew list to check what installed now